tests/integration/components/signup-steps/step-one-test.js

```diff
@@ -0,0 +1,79 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'website-www/tests/helpers';
+import { render, typeIn, fillIn } from '@ember/test-helpers';
+import { set } from '@ember/object';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | signup-steps/step-one', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('render firstname input field', async function (assert) {
+    assert.expect(14);
+    this.set('name', 'firstname');
+    this.set('field', 'First Name');
+    this.set('placeHolder', 'Write your first name');
+    this.set('type', 'text');
+    this.set('required', true);
+    this.set('value', '');
+    this.set('disabled', false);
+    this.set('onInput', (e) => {
+      // Update the input value when the input changes
+      this.value = e.target.value;
+      // Call handleInputChange with the current value
+      this.handleInputChange('firstname', this.value);
+    });
+
+    this.set('handleInputChange', (actual1, actual2) => {
```

Comment:
Why actual1 and actual2?
In my opinion, this doesn't make clear sense of what it actually is.

Reply:
Done with changes can you please check my latest commit